exercises/collatz-conjecture/package.yaml

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
extra newline at the beginning of this file, it appears? I would remove it.

Reply:
(it has been removed, now github put my comment on the wrong line, so I'll delete this comment)